### PR TITLE
Add Perl Support

### LINKED ISSUE #1: Would you consider adding perl support?
http://serverfault.com/a/247358/116303

This would allow me to capture my linked containers' IP addresses using something like `perl_set $ghost 'sub { return $ENV{"GHOST_PORT_2368_TCP_ADDR"}; }';`, which I can then use in `upstream`/`server` placement.

### PR DESCRIPTION
**Pull request by johnjelinek**
Example:

Adding this line to my `nginx.conf` file does not produce an error:

```
perl_set  $rnd 'sub { return int(rand(10 - 5)) + 5; }';
```

Size before:

```
$ docker images
REPOSITORY           TAG                 IMAGE ID            CREATED             VIRTUAL SIZE
nginx                latest              bc32c7487f40        23 minutes ago      444.1 MB
```

Size after:

```
$ docker images
REPOSITORY           TAG                 IMAGE ID            CREATED             VIRTUAL SIZE
nginx                latest              32293f4d9c89        19 seconds ago      458.4 MB
```

Fixes #1

### CODE REVIEW

**Comment by yosifkit** [comment]
This looks great to me.  Thanks!

cc @tianon

File: Dockerfile [hunk 1]
```diff
@@ -2,11 +2,13 @@ FROM debian:jessie
...
 RUN apt-get update && apt-get install -y \
 		build-essential \
+		perl \
 		libgd2-dev \
 		libgeoip-dev \
 		libpcre3-dev \
 		libssl-dev \
-		libxslt1-dev
+		libxslt1-dev \
+		libperl-dev
```
Comment (tianon) [inline Dockerfile]: I'm being a little pedantic, but would you mind sorting these alphabetically? :)

**Comment by tianon** [comment]
Barring my overextreme pedantism, LGTM. :)

**Comment by johnjelinek** [comment]
@tianon: will do, but I recommend creating a Contributing Guidelines doc for future reference.

**Comment by johnjelinek** [comment]
Updated, please review @tianon.

**Comment by tianon** [comment]
:heart: thanks

LGTM

**Comment by tianon** [comment]
I've added a note about the list being sorted in d2c7eee53b81ae70f4eb9e3348575b35df136c95. :)

**Comment by johnjelinek** [comment]
:+1:

**Comment by johnjelinek** [comment]
Any idea when this will appear in the docker index? `docker pull nginx` didn't seem to grab the latest.

**Comment by tianon** [comment]
Oh shoot @johnjelinek, I'm so sorry.  I merged and completely forgot to PR against stackbrew to update the Hub.  I'll do that now.

**Pull request merged**